docs/api/structures/printer-info.md

```diff
@@ -1,9 +1,11 @@
 # PrinterInfo Object
 
-* `name` String
-* `description` String
-* `status` Number
-* `isDefault` Boolean
+* `name` String - the name of the printer as understood by the OS.
+* `displayName` String - the name of the printer as shown in Print Preview.
+* `description` String - a longer description of the printer's type.
+* `status` String - the current status of the printer. Can be 'idle', 'paused', 'in use', 'offline', or 'unknown'.
```

Comment:
is changing the status from a number to a string a breaking change?

Reply:
On principle yes - but i would also say that the previous output was essentially unintelligible in that we neither described what 'status' meant nor provided a way to interpret it.

Reply:
Ha, fair enough. It seems like an improvement, I agree. Is it small enough of a change that we can skip putting it in the breaking changes doc?

Reply:
(resolved, no longer applicable!)